packages/client-sdk-nodejs/src/internal/grpc/retry-interceptor.ts

```diff
@@ -116,23 +122,24 @@ export class RetryInterceptor {
                 //
                 // We also need this check in case DEADLINE_EXCEEDED is marked as a retryable status code
                 // as it is in the default storage eligibility strategy.
-                if (new Date(Date.now()) >= overallDeadline) {
+                if (getCurrentTimeAsDateObject() >= overallDeadline) {
```

Comment:
can replace this and the other instances with something like `hasExceededDeadlineRelativeToNow(overallDeadline)` or some better now.